modules/react/pagination/lib/Pagination/GoTo/TextInput.tsx

```diff
@@ -10,11 +11,25 @@ export type GoToTextInputProps = TextInputProps & {
   value?: string | number;
 };
 
+export const paginationGoToTextInputStencil = createStencil({
```

Comment:
Same here, since this renders a TextInput, do we want to extend text input stencil?